runtime/v2/bundle.go

```diff
@@ -121,7 +121,7 @@ func (b *Bundle) Delete() error {
 	if err := mount.UnmountAll(rootfs, 0); err != nil {
```

Comment:
Checked that this already ignores non-existing paths; https://github.com/containerd/containerd/blob/433662502f428773746fe403484d61452e77f7e2/mount/mount_linux.go#L157